core/src/main/resources/db/migration.sql

```diff
@@ -0,0 +1,14 @@
+##version: 1.0.0
+CREATE TABLE info (DB_SCHEMA_VERSION VARCHAR(8));
+INSERT INTO info VALUES ("1.0.0");
+
+##version: 1.1.0
+CREATE TABLE sample_list LIKE patient_list;
+INSERT sample_list SELECT * FROM patient_list;
+CREATE TABLE sample_list_list LIKE patient_list_list;
+INSERT sample_list_list SELECT * FROM patient_list_list;
+DROP TABLE micro_rna;
```

Comment:
@jjgao : regarding your last comment to not have drop statements, do you want to have another procedure for this? Just wondering you are planning to handle this scenario.